### ISSUE TITLE
wasm version of zxing for web

### ISSUE THREAD

**Issue by alexandrim0** [juliansteenbakker/mobile_scanner#1231]
Hello!

Thank you for this great package!

Would you plan to use wasm version of zxing for web?

May be [this](https://github.com/Sec-ant/zxing-wasm)

**Comment by navaronbracke** [comment]
Thanks for the link. Currently, our WASM support still has some issues with the video preview. (seems to be a bug with platform views?) and we didn't test much of it because of that.

We can definitely look into using the WASM version of the library, though.
I will mark this as a feature request.

Out of curiosity, do you know if this WASM version has better maintenance then the old ZXing JS version?
The JS version seems to have stopped feature development.

**Comment by alexandrim0** [comment]
Yes, I can see the issue with the video preview. Actually, this is one only bug which I can not fix and be happy with my wasm compiled app.

That repository looks very fresh and popular. In my own belief, wasm much more perspective than js for web when we talking about software like zxing.

It could be, guys from zxing-js think same way and that is why their "Project in Maintenance Mode Only".

**Comment by wkok** [comment]
> _Yes, I can see the issue with the video preview. Actually, this is one only bug which I can not fix and be happy with my wasm compiled app._

Same here. Would be great if we could get the preview working on wasm.

**Comment by navaronbracke** [comment]
I haven't had the time to file a repro for the preview being misaligned. Feel free to strip down the example app to only the video element (so nothing with zxing) to try and repro the alignment issue and file a bug on flutter/flutter for that.

Otherwise I'll go do that in the next few days.

**Comment by juliansteenbakker** [comment]
See #1664 for wasm support
